{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py

```diff
@@ -72,6 +73,8 @@
 
 # Database
 # https://docs.djangoproject.com/en/1.9/ref/settings/#databases
+if not config('DATABASE_URL').startswith('sqlite') and not wait_for_url(config('DATABASE_URL')):
```

Comment:
@pmac is this the way you intended to use `urlwait`? Also is there another way to work around `sqlite` urls?